### ISSUE TITLE
Implement rendering of HTML in mod descriptions

### ISSUE THREAD

**Issue by joseasoler** [RimSort/RimSort#141]
**Is your feature request related to a problem? Please describe.**

Consider the following code, when used in the `description` field of the About.xml file of a mod:

```xml
<description><![CDATA[<size=18><b><color=teal>Pathfinding Framework</color></b></size>

Enables the creation of custom movement rules for pawns.

This mod is an overhaul of TerrainMovementKit and SwimmingKit, with a focus on performance and mod compatibility.
]]></description>
```

This is how it looks in-game:

![imagen](https://github.com/RimSort/RimSort/assets/3092211/c8101a05-5677-4e61-9f71-4318dc3c7eee)

And this is how it currently looks in RimSort:

![imagen](https://github.com/RimSort/RimSort/assets/3092211/0300874a-11a0-4e01-999d-ff0508f3b131)

Here is another example, without using CDATA. In this case it is necessary to escape the `<` and `>` characters using `&lt;` and `&gt;` respectively.

```xml
<description>&lt;size=24&gt;JoyKind: Condensed&lt;/size&gt;

Combined version of all 5 JoyKind modules - Alcohol, Ambrosia, Caffeine, Smokeleaf, and Tobacco.

Adds each chemical as its own JoyKind, with a new JoyGiver, to separate them as a recreation item from drugs. 

WARNING: These drugs no longer satisfies the Chemical need of Chemical Interest or Fascination.

</description>
```

![imagen](https://github.com/RimSort/RimSort/assets/3092211/55050bdc-3350-40b2-baba-cb4d4d575523)

![imagen](https://github.com/RimSort/RimSort/assets/3092211/6affdfff-18e8-4f6d-8aaa-824e7d976570)

**Describe the solution you'd like**

Match how the descriptions look in-game and in RimSort by rendering HTML tags in the RimSort description text box.

**Comment by jefferyharrell** [comment]
Do we have any sense for how many mods, as a fraction, use HTML markup in their descriptions? If it's like _two_ then I'm less enthusiastic about this suggestion than if it's like 40%. Just asking to ask.

**Comment by joseasoler** [comment]
Far less than that. In my modlist it is 12 mods out of 372. I don't think this task should be prioritized over more pressing feature work, but I still believe this task could be a good candidate for a "help wanted" and/or "good first issue" labels.

**Comment by jefferyharrell** [comment]
I like the sound of that. What do we think about Markdown in the description attribute? Is that a thing people do?

**Comment by joseasoler** [comment]
I don't think RimWorld can show markdown syntax in the mods list That would lead to different representations in RimSort and in the game.

**Comment by hk21702** [comment]
This is implemented in #571 

RimWorld is actually using this weird format made specifically for unity that needs to be converted to HTML for PySide6/Qt to handle it. 

https://docs.unity3d.com/Manual/UIE-rich-text-tags.html
https://docs.unity3d.com/Packages/com.unity.ugui@1.0/manual/StyledText.html
